parsely/src/test/java/com/parsely/parselyandroid/EventsBuilderTest.kt

```diff
@@ -0,0 +1,216 @@
+package com.parsely.parselyandroid
+
+import android.content.Context
+import android.provider.Settings
+import androidx.test.core.app.ApplicationProvider
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.MapAssert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+internal class EventsBuilderTest {
+
+    private lateinit var sut: EventsBuilder
+
+    @Before
+    fun setUp() {
+        val applicationContext = ApplicationProvider.getApplicationContext<Context>()
+        sut = EventsBuilder(
+            applicationContext,
+            TEST_SITE_ID,
+        )
+        Settings.Secure.putString(
+            applicationContext.contentResolver,
+            Settings.Secure.ANDROID_ID,
+            "android_id"
+        )
+    }
+
+    @Test
+    fun `events builder prepares correct pageview pixel`() {
```

Comment:
Thanks! I've updated the names of tests, but decided to leave comments. For some reason, they make the test much more readable for me, the space feels easy to misplace by accident. I'll keep thinking about this though :) 

>  I notice you are not including the then, why is that? 

Did you mean name of tests? Because every test contain the `then` part - or am I missing something?

Reply:
> Thanks! I've updated the names of tests, but decided to leave comments. For some reason, they make the test much more readable for me, the space feels easy to misplace by accident.

Yes, I understand this @wzieba , it is indeed helping if you aren't used to just depend on line breaks.

> I'll keep thinking about this though :)

🤔 🥇 😄 

> Did you mean name of tests? Because every test contain the then part - or am I missing something?

Me being... 🔍 🔍 🔍 😄 

An example of such test is this one: `given extraData is null, when creating a pixel, don't include extraData`

Notice the `then` missing from the last sentence which starts with `don't` instead of `then don't`. 🤷

Reply:
Gotcha, so it was the name - updated 🙌